app/uk/gov/hmrc/brm/services/MatchingService.scala

```diff
@@ -17,24 +17,36 @@
 package uk.gov.hmrc.brm.services
 
 import uk.gov.hmrc.brm.audit.{BRMAudit, EnglandAndWalesAuditEvent}
+import uk.gov.hmrc.brm.config.BrmConfig
 import uk.gov.hmrc.brm.models.brm.Payload
-import uk.gov.hmrc.brm.models.response.gro.GroResponse
 import uk.gov.hmrc.brm.models.matching.ResultMatch
+import uk.gov.hmrc.brm.models.response.Record
 import uk.gov.hmrc.brm.utils.BrmLogger._
 import uk.gov.hmrc.brm.utils.MatchingType
+import uk.gov.hmrc.play.config.ServicesConfig
 import uk.gov.hmrc.play.http.HeaderCarrier
 
-trait MatchingService {
+trait MatchingService extends ServicesConfig {
   val CLASS_NAME: String = this.getClass.getCanonicalName
 
-  def performMatch(input: Payload, response: GroResponse, matchingType: MatchingType.Value)(implicit hc: HeaderCarrier) : ResultMatch = {
+  val matchOnMultiple: Boolean
+
+  def performMatch(input: Payload,
+                   records: List[Record],
+                   matchingType: MatchingType.Value)
+                  (implicit hc: HeaderCarrier): ResultMatch = {
+
     info(CLASS_NAME, "MatchingType", s"$matchingType")
+
     val algorithm = matchingType match {
       case MatchingType.FULL => FullMatching
       case MatchingType.PARTIAL => PartialMatching
     }
 
-    val result = algorithm.performMatch(input, response)
+    val result = algorithm.performMatch(input, records, matchOnMultiple)
+
+    if (result.isMatch) BRMAudit.logEventRecordFound(hc)
```

Comment:
Remove this for the splunk auditing as this would not audit for this metric as the match would be false